_posts/13-03-01-Docker.md

```diff
@@ -5,41 +5,36 @@ anchor:  docker
+
+Docker may help if you're developing multiple websites and want the separation that comes from installing each on it's on virtual machine, but don't have the necessary disk space or the time to keep everything up to date.  It's efficient: the installation and downloads are quicker, you only need to store one copy of each image however often it's used, containers need less RAM and share the same OS kernel, so you can have more servers running simultaneously, and it takes a matter of seconds to stop and start them, no need to wait for a full server boot. 
 
 ### Example: Running your PHP Applications in Docker
 
-After you [installed docker][docker-install] on your machine, you can start an Apache with PHP support in one step.
-The following command will download a fully functional Apache installation with the latest PHP version and provide the
-directory `/path/to/your/php/files` at `http://localhost:8080`:
+After [installing docker][docker-install] on your machine, you can start a web server with one command.
+The following will download a fully functional Apache installation with the latest PHP version, map `/path/to/your/php/files` to the document root, which you can view at `http://localhost:8080`:
 
 {% highlight console %}
 docker run -d --name my-php-webserver -p 8080:80 -v /path/to/your/php/files:/var/www/html/ php:apache
 {% endhighlight %}
 
-After running `docker run` your container is initialized and running.
-If you would like to stop or start your container again, you can use the provided name attribute and simply run
-`docker stop my-php-webserver` and `docker start my-php-webserver` without providing the above mentioned parameters
-again.
+This will initialize and launch your container. `-d` makes it runs in the background. To stop and start it, simply run `docker stop my-php-webserver` and `docker start my-php-webserver` (the other parameters are not needed again).
 
 ### Learn more about Docker
 
-The commands mentioned above only show a quick way to run an Apache web server with PHP support but there are a lot
-more things that you can do with Docker. One of the most important things for PHP developers will be linking your
-web server to a database instance, for example. How this could be done is well described within the [Docker User Guide]
-[docker-doc].
+The command above shows a quick way to run a basic server.  There's much more you can do (and thousands of pre-built images in the [Docker Hub][docker-hub].) Take time to learn the terminology and read the [Docker User Guide][docker-doc] to get the most from it, and don't run random code you've downloaded without checking it's safe – unofficial images may not have the latest security patches. If in doubt, stick to the [official repositiories][docker-hub-official].
```

Comment:
> There's much more you can do (and thousands of pre-built images in the [Docker Hub][docker-hub].)

This may sound really picky, but the closing `.` at the end of the sentence should be outside the parentheses.